src/ODEs/TransientFETools/TransientFEOperators.jl

```diff
@@ -436,10 +436,101 @@ function explicit_rhs!(
   explicit_rhs
 end
 
+
+# EX-RK Transient FE operators
+"""
+Used in Explicit Runge-Kutta schemes
+"""
+struct TransientEXRKFEOperatorFromWeakForm{C} <: TransientFEOperator{C}
+  res::Function
+  lhs::Function
+  rhs::Function
+  jacs::Tuple{Vararg{Function}}
+  assem_t::Assembler
+  trials::Tuple{Vararg{Any}}
+  test::FESpace
+  order::Integer
+end
+
+
+function TransientEXRungeKuttaFEOperator(lhs::Function,rhs::Function,jac::Function,
+  jac_t::Function,trial,test)
+  res(t,u,v) = lhs(t,u,v) - rhs(t,u,v)
+  assem_t = SparseMatrixAssembler(trial,test)
+  TransientEXRKFEOperatorFromWeakForm{Nonlinear}(res,lhs,rhs,(jac,jac_t),assem_t,(trial,∂t(trial)),test,1)
```

Comment:
Why nonlinear? I think that for a explicit method you just need a linear operator, that represents the mass matrix

Reply:
There is no linear OperatorType in ODEOperators.jl. Of the OperatorTypes included, I think Nonlinear is the best choice.